flybywiresim/a32nx
# fix(nd): VOR/ADF info after Asobo update 1.14.5.0

Fixes #3880

## Summary of Changes

The signature of `Simplane.getAutoPilotNavAidState` changed, with addition of aircraft type as the first parameter. We use this to determine the status of the ADF/OFF/VOR switches.



## Screenshots (if necessary)


![image](https://user-images.githubusercontent.com/9995998/110573431-20dd2580-81c0-11eb-9ced-e37d7c038370.png)

## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

- Check that VOR/ADF selector switches operate correct side on each ND.
- Check VOR and ADF displays okay on bottom corners.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
